src/main/java/bio/terra/service/auth/ras/ECMService.java

```diff
@@ -1,23 +1,46 @@
 package bio.terra.service.auth.ras;
 
 import bio.terra.app.configuration.ECMConfiguration;
+import bio.terra.common.iam.AuthenticatedUserRequest;
+import bio.terra.externalcreds.api.OidcApi;
 import bio.terra.externalcreds.api.PassportApi;
 import bio.terra.externalcreds.client.ApiClient;
 import bio.terra.externalcreds.model.ValidatePassportRequest;
 import bio.terra.externalcreds.model.ValidatePassportResult;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jwt.JWTClaimsSet;
+import com.nimbusds.jwt.JWTParser;
+import java.text.ParseException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+import liquibase.util.StringUtils;
+import org.apache.commons.collections4.ListUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
 import org.springframework.web.client.RestTemplate;
 
 @Service
 public class ECMService {
+  private final Logger logger = LoggerFactory.getLogger(ECMService.class);
```

Comment:
I think it's preferred to write this as `static`
```suggestion
  private static final Logger logger = LoggerFactory.getLogger(ECMService.class);
```
```suggestion
  private final Logger logger = LoggerFactory.getLogger(ECMService.class);
```